pkgs/development/compilers/emscripten/default.nix

```diff
@@ -1,17 +1,17 @@
 { lib, stdenv, fetchFromGitHub, python3, nodejs, closurecompiler
 , jre, binaryen
-, llvmPackages_11
-, symlinkJoin, makeWrapper
+, llvmPackages_13
+, symlinkJoin, makeWrapper, substituteAll
 , mkYarnModules
 }:
 
 stdenv.mkDerivation rec {
   pname = "emscripten";
-  version = "2.0.10";
+  version = "2.0.26";
 
   llvmEnv = symlinkJoin {
     name = "emscripten-llvm-${version}";
-    paths = with llvmPackages_11; [ clang-unwrapped lld llvm ];
+    paths = with llvmPackages_13; [ clang-unwrapped clang-unwrapped.lib lld llvm ];
```

Comment:
Maybe we should use llvmPackages and override it.

Reply:
done